source/includes/webhooks/_introduction.md

```diff
@@ -188,14 +188,14 @@ If a webhook is disabled, it will not trigger when the event occurs. Webhooks be
 
 ## Retry policy
 
-In the event of a failed webhook request (due to timeout, a non HTTP 200 response, or network issues), Greenhouse will attempt a maximum of 6 retries according to the formula on the right:
+In the event of a failed webhook request (due to timeout, a non HTTP 200 response, or network issues), Greenhouse will make up to 6 attempts according to the formula on the right.
 
 This formula increases the amount of time between each retry, while assigning a random number of seconds to avoid consistent failures from overload or contention.
 
-Greenhouse will attempt 6 retries over the course of 15 hours.
+Greenhouse will make up to 6 attempts over the course of 7 hours.
 
 ```ruby
```

Comment:
Do we even need this part? I would suggest removing the code section.

Reply:
Thanks, done